task/transcode.go

```diff
@@ -1,342 +1,13 @@
 package task
 
 import (
-	"bytes"
-	"context"
-	"errors"
-	"fmt"
-	"io"
 	"math/rand"
-	"os"
-	"sync/atomic"
 	"time"
 
-	"github.com/golang/glog"
-	api "github.com/livepeer/go-api-client"
-	"github.com/livepeer/joy4/av"
-	"github.com/livepeer/joy4/av/avutil"
 	"github.com/livepeer/joy4/format"
-	"github.com/livepeer/joy4/format/mp4"
-	"github.com/livepeer/joy4/format/ts"
-	"github.com/livepeer/livepeer-data/pkg/data"
-	"github.com/livepeer/stream-tester/model"
-	"github.com/livepeer/stream-tester/segmenter"
-)
-
-const (
-	segLen               = 2 * time.Second
-	maxFileSizeForMemory = 50_000_000
 )
 
 func init() {
 	format.RegisterAll()
 	rand.Seed(time.Now().UnixNano())
```

Comment:
@victorges Any idea why this was here? It's a deprecated function anyway

Reply:
I think this was probably copied from the record-tester code that this is based of: https://github.com/livepeer/stream-tester/blob/9b92a073ee2dd84323f0c8c8686458c3319b33c0/cmd/recordtester/recordtester.go#L50C34-L50C34

It was included in the first PR for this https://github.com/livepeer/task-runner/pull/7/files#diff-d09eb25053ed0ed141e44c9262f4138cb013dd9b1a386428d27f9366c0e44487R32 without much explanation nor use of `rand` package anywhere else, so I think it might've been a miscopy